src/main.ts

```diff
@@ -39,7 +40,9 @@ async function bootstrap() {
   app.use(helmet());
   app.enableCors();
 
-  await app.listen(3000);
+  const configService = app.get(ConfigService);
+  const port = configService.get<number>('PORT');
+  await app.listen(port);
```

Comment:
Please add callback and logger with message "Server listen on http://{host}:{port)"